doc/man/8/radosgw-admin.rst

```diff
@@ -323,44 +543,164 @@ Options
 
 .. option:: --purge-data
 
-   Remove user data before user removal.
+   When specified, user removal will also purge all the user data.
 
 .. option:: --purge-keys
 
 	When specified, subuser removal will also purge all the subuser keys.
    
 .. option:: --purge-objects
 
-   Remove all objects before bucket removal.
+   When specified, the bucket removal will also purge all objects in it.
 
 .. option:: --metadata-key=<key>
 
 	Key to retrieve metadata from with ``metadata get``.
 
+.. option:: --remote=<remote>
+
+   Zone or zonegroup id of remote gateway.
+
+.. option:: --period=<id>
+
+   Period id.
+
+.. option:: --epoch=<number>
+
+   Period epoch.
+
+.. option:: --commit
+
+   Commit the period during 'period update'.
+
+.. option:: --staging
+
+   Get the staging period info.
+
+.. option:: --master
+
+   Set as master.
+
+.. option:: --master-zone=<id>
```

Comment:
missing --master-url and --master-zonegroup

Reply:
These were already removed in rgw_admin.cc.